javascript/code-challenges/challenges-06.test.js

```diff
@@ -0,0 +1,190 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1 - Review
+
+Write a function named getNames that, given an array of people objects, uses map to return an array of names reversed.
+
+For example:
+[
+{
+  name: 'lloyd',
+  age: 32,
+  shoeSize: 12
+},
+{
+  name: 'jamie',
+  age: 21,
+  shoeSize: 8
+}
+]
+
+Returns: ['dyoll', 'eimaj'];
+------------------------------------------------------------------------------------------------ */
+
+const getNames = (arr) => {
+  // Solution code here...
+  const reversed = arr.map((names) => {
+    let objNames = names.name;
```

Comment:
this is kind of superfluous. You can just return names.name.split. Or, what I did, what use the word obj. That helped me with clarity.
```js
return arr.map(obj => obj.name.split('').reverse().join('');
```